recipes/datapane/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "datapane" %}
+{% set version = "0.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 398fdf46259e6e7485e01ddb817fe27158797120346464fbe476c80ac13f8e5b
+  
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<36]
+  entry_points:
+    - datapane = datapane.client.__main__:main
+    - dp-runner = datapane.runner.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - poetry
+  run:
+    - python
+    - dataclasses  # [py==36]
```

Comment:
```suggestion
    - dataclasses
```

Reply:
`dataclasses` is built into the standard library on python 3.7+ - will this cause a problem?